adapter/pkg/messaging/notificationListener.go

```diff
@@ -44,14 +44,24 @@ const (
 	applicationRegistration     = "APPLICATION_REGISTRATION_CREATE"
 	removeApplicationKeyMapping = "REMOVE_APPLICATION_KEYMAPPING"
 	apiLifeCycleChange          = "API_LIFECYCLE_CHANGE"
+	applicationCreate           = "APPLICATION_CREATE"
+	applicationUpdate           = "APPLICATION_UPDATE"
+	applicationDelete           = "APPLICATION_DELETE"
+	subscriptionCreate          = "SUBSCRIPTIONS_CREATE"
+	subscriptionUpdate          = "SUBSCRIPTIONS_UPDATE"
+	subscriptionDelete          = "SUBSCRIPTIONS_DELETE"
+	policyCreate                = "POLICY_CREATE"
+	policyUpdate                = "POLICY_UPDATE"
+	policyDelete                = "POLICY_DELETE"
 )
 
 // var variables
 var (
-	APIList                  = make([]resourceTypes.API, 0)
-	ScopeList                = make([]resourceTypes.Scope, 0)
-	APIListTimeStamp         = make(map[string]int64, 0)
-	ApplicationListTimeStamp = make(map[string]int64, 0)
+	ScopeList                         = make([]resourceTypes.Scope, 0)
+	apiListTimeStampMap               = make(map[string]int64, 0)
+	subsriptionsListTimeStampMap      = make(map[string]int64, 0)
```

Comment:
Comment the reason behind keeping the timestamp map

Reply:
fixed via https://github.com/wso2/product-microgateway/pull/1651/commits/34135e565e0feb892396850af240c35fdc8c4499